test/blackbox-tests/test-cases/pkg/helpers.sh

```diff
@@ -50,3 +50,17 @@ cat >dune.lock/lock.dune <<EOF
 (lang package 0.1)
 EOF
 }
+
+# [mkenvtestpkg <package> <depends> <env_update_op>] makes an opam <package> that depends
+# on <depends> and updated "env_var" using <env_update_op>
+mkenvtestpkg() {
+  mkpkg $1 <<EOF
+opam-version: "2.0"
+depends: [ "${2:-}" ]
```

Comment:
How does the default value for `$2` help here?

Reply:
Thats the package that this one will depend on. We have 3 of them in the tests: package1->package2->package3. We are testing how setenv affects the env of the next package.

Reply:
Originally this was a 'EOF' so passing "" wouldn't be empty. I've removed it since so Ill remove the default value too.